### PR TITLE
Animated logo

### PR DESCRIPTION
**Pull request by lpoulter**
Credit to @josa42 who did the vast bulk of this work.

Intended to be used in specific products. Brandwatch logo slides away to
reveal Product initials

### CODE REVIEW

**Review by lcapel — APPROVED** [review]
Sweet 👍 even has support for custom svg 👍 What was your process for creating the additional svg?

File: packages/axiom-components/src/AnimatedLogo/AnimatedLogo.css [hunk 1]
```diff
@@ -0,0 +1,235 @@
+/* Initial state: open */
+.animated-logo {
```
Comment (josa42) [inline packages/axiom-components/src/AnimatedLogo/AnimatedLogo.css:2]: Just out of interest, should the classes be prefixed `ax-`?

**Review by josa42 — APPROVED** [review]
👍🏻 Very cool to see this going into axiom!

File: packages/axiom-components/src/AnimatedLogo/AnimatedLogo.css (hunk 1 above)
Comment (lpoulter) [reply packages/axiom-components/src/AnimatedLogo/AnimatedLogo.css:2]: Yes they damn well should, good spot.

**Pull request merged**